### DISCUSSION TITLE
Dumb shuffle mode

### DISCUSSION THREAD

**Discussion by owenoliver1** [tidal-music/discussions#76] [Ideas & Feature requests]
Can you please add a dumb shuffle mode with no logic for the shuffle button other than not repeating the same song.

It just gets a list of all of the songs and runs RAND(SONGID) for the playlist to stop it playing the same songs over and over.

This is a common complaint from people using Spotify and I have a playlist with 586 tracks and when I hear the same songs each week more than once I know its trying to be smart with the shuffle but I like all the songs on the playlist.

**Comment by stefanmoro** [comment]
Hi @owenoliver1 ,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API/SDK, open source projects etc).

As this question relates to our existing product offering, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
